app/src/main/res/values-bn-rBD/strings.xml

```diff
@@ -675,7 +675,7 @@ Scanning will happen automatically.</string>
   <string name="verification_codes">Verification codes</string>
   <string name="premium_subscription_required">Premium subscription required</string>
   <string name="cannot_add_authenticator_key">Cannot add authenticator key?</string>
-  <string name="scan_qr_code">Scan QR Code</string>
+    <string name="scan_qr_code">Scan QR Code</string>
```

Comment:
Why are some of these indented?

Reply:
They're coming from Crowdin like this for some reason. I've not identified a pattern to it. Seems like every time I pull down translations it indents random strings in random files. I figure once I've pulled down enough times they'll all be fixed. 😅